src/shared_modules/indexer_connector/tests/unit/monitoring_test.cpp

```diff
@@ -120,7 +120,7 @@ TEST_F(MonitoringTest, TestInvalidServer)
     // Interval to check the health of the servers
     std::this_thread::sleep_for(std::chrono::seconds(MONITORING_HEALTH_CHECK_INTERVAL + 5));
 
-    EXPECT_STREQ(logMessage.c_str(), "Couldn't connect to server. Status code: -1");
```

Comment:
Note: New curl dependency changed this exception (see lib/strerror.c file)  in [this commit](https://github.com/curl/curl/commit/c074ba64a86c36bbb697b89e1b3509fd14d12af3)

Before:
```C++
  case CURLE_COULDNT_CONNECT:
    return "Couldn't connect to server";
```

After:
```C++
  case CURLE_COULDNT_CONNECT:
    return "Could not connect to server";
```